appConfig/appConfig.snd3.yaml

```diff
@@ -21,3 +21,16 @@
 - key: COOKIE_PASSWORD
   value: "{{servicename}}-COOKIE-PASSWORD"
   type: "keyvault"
+- key: MESSAGE_HOST
+  value: "{{servicename}}-MESSAGE-HOST"
+  type: "keyvault"
+- key: MESSAGE_USER
+  value: "{{servicename}}-MESSAGE-USER"
+  type: "keyvault"
+- key: MESSAGE_PASSWORD
+  value: "{{servicename}}-MESSAGE-PASSWORD"
+  type: "keyvault"
```

Comment:
I don't think you need these, when deployed to ADP it will use a managed identity rather than a user name and password.

ie, `useCredentialChain` should be set to `true` in that scenario so the configuration should reflect that.  (or do like we have in the other services.  if `NODE_ENV === production` then use credential chain for service bus authentication.

Reply:
Same as previous comment as this would also be part of the config component.

Reply:
**Update:** These variables have now been removed.